src/Components/Profile/Profile.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable camelcase */
+
+import { useSelector } from 'react-redux';
+
+const Profile = () => {
+  const missions = useSelector((state) => state.missions);
+  const rockets = useSelector((state) => state.rockets);
+  const rocketsList = rockets.filter((r) => r.reserved === true).map((rocket) => (<li key={rocket.id} className="list-group-item">{rocket.name}</li>));
+  const list = missions.filter((r) => r.joined === true).map((mission) => (
```

Comment:
The `filter()` method is used to render the reserved rockets and missions 🚀 Fantastic 🙌

Reply:
@mateo951 Thank you for the approval